.vale.ini

```diff
@@ -17,7 +17,7 @@ Vale.Avoid = YES
 # Ignore files in dirs starting with `.` to avoid raising errors for `.vale/fidns_ndotsures/*/testinvalid.adoc` files
 [[!.]*.adoc]
 
-BasedOnStyles = RedHat, AsciiDoc
+BasedOnStyles = RedHat, AsciiDoc, AsciiDocDITA
```

Comment:
I recommend you not try combining all 3 packages. Try a PR with just AsciiDocDITA. I suspect we will need to run it as a separate GitHub workflow.